noetic/distribution.yaml

```diff
@@ -9118,6 +9118,28 @@ repositories:
       url: https://github.com/ros-geographic-info/unique_identifier-release.git
       version: 1.0.6-1
     status: maintained
+  uos_tools:
+    doc:
+      type: git
+      url: https://github.com/uos/uos_tools.git
+      version: master
+    release:
+      packages:
+      - uos_common_urdf
+      - uos_diffdrive_teleop
+      - uos_freespace
+      - uos_gazebo_worlds
+      - uos_maps
+      - uos_tools
```

Comment:
@spuetz Do you know which license `uos_tools` is?

@clalancette if a package is dual licensed, what should the LICENSE file in the repo contain?

Reply:
I don't think we have a specific prescription.  Some repositories host both licenses in the same file, some have the licenses in separate files (like `LICENSE.bsd` and `LICENSE.mit`, etc), some have it in the individual packages (so package A has LICENSE with BSD, package B has LICENSE with MIT, etc).

As long as it is clearly communicated, I don't think it matters too much which path is chosen.

Reply:
It would probably be best to simplify the metapackage w/o content to be BSD like the rest of the system and clarify specifically the package with GPL code in it specifically as such.